base/shared/src/main/scala/scalaz/data/MaybeSyntax.scala

```diff
@@ -2,5 +2,14 @@ package scalaz
 package data
 
 trait MaybeSyntax {
-  implicit class OptionAsMaybe[A](oa: Option[A]) { def asMaybe: Maybe[A] = Maybe.fromOption(oa) }
+  implicit final class OptionAsMaybe[A](oa: Option[A]) { def asMaybe: Maybe[A] = Maybe.fromOption(oa) }
+
+  implicit final class ToMaybeOps[A](a: A) {
+    def just: Maybe[A] = Maybe.just(a)
+  }
+}
+
+trait MaybePrelude {
+  def empty[A] = Maybe.empty[A]
+  def just[A](a: A) = Maybe.just(a)
```

Comment:
`MaybeFunctions` served as a common interface of `MaybeModule` and `MaybePrelude`. If `MaybePrelude` is not going to extend `MaybeFunctions` anymore, then I think it is better (for clarity) to remove `MaybeFunctions` and move its content to `MaybeModule`.

Reply:
👍 That way I can rename `MaybePrelude` to `MaybeFunctions` and have all the naming consistent.